lib/stm32/common/exti_common_all.c

```diff
@@ -57,58 +84,256 @@ void exti_set_trigger(uint32_t extis, enum exti_trigger_type trig)
 		EXTI_FTSR |= extis;
 		break;
 	}
+#endif
+	
 }
 
+
+/*---------------------------------------------------------------------------*/
+/** @brief Set trigger for [63:32] extended interrupts.
+
+Set trigger values  extended [63:32] interrupts
+
+@param[in] extis extended interrupt number [63:32]  @ref exti_values.
+@param[in] trig extended interrupt front type @ref exti_trigger_values.
+*/
+
+void exti_set_trigger2(uint32_t extis, enum exti_trigger_type trig)
+{
+	
+#if defined(EXTI_RTSR2) && defined(EXTI_FTSR2)
+	switch (trig) {
+	case EXTI_TRIGGER_RISING:
+		EXTI_RTSR2 |= extis;
+		EXTI_FTSR2 &= ~extis;
+		break;
+	case EXTI_TRIGGER_FALLING:
+		EXTI_RTSR2 &= ~extis;
+		EXTI_FTSR2 |= extis;
+		break;
+	case EXTI_TRIGGER_BOTH:
+		EXTI_RTSR2 |= extis;
+		EXTI_FTSR2 |= extis;
+		break;
+	}
+#else
+	exti_set_trigger(extis, trig);
```

Comment:
they shoulnd't be able to _call_ exti2 functions without the definitions, not like this please.